### PR TITLE
Make featured story button round and add a little spacing to buttons in mission statement

### PR DESCRIPTION
**Pull request by flora8984461**
Fix https://github.com/EclipseFdn/ecdtools.eclipse.org/issues/71#issuecomment-734267107

Signed-off-by: Yi Liu <yi.liu@eclipse-foundation.org>

### CODE REVIEW

File: less/styles.less [hunk 1]
```diff
@@ -21,6 +21,10 @@
   background-position: 0 100%;
   .featured-jumbotron{
     padding-top:48px;
+
+    .btn {
+      border-radius: 1em;
```
Comment (ericpoirier) [inline less/styles.less:26]: Instead of updating the border radius manually in CSS you should update Bootstrap's variable: @border-radius-base.

Comment (flora8984461) [reply less/styles.less:26]: it's already here
https://github.com/flora8984461/ecdtools.eclipse.org/blob/yiliu/master/makeFeaturedStoryButtonRound/less/_variables.less#L37

But for `.featured-jumbotron. btn` it has `border-radius: 0` on small size screen, I think there is no variable for it? 🤔

Comment (flora8984461) [reply less/styles.less]: I update it to use the variable in the styles.less instead of "1em"

Comment (ericpoirier) [reply less/styles.less]: Ah ok I didn't think we were overriding it for the Jumbotron. No I don't think we would have created a variable for that. I would've thought that all the buttons on the website would have the same border radius for consistency?

Comment (flora8984461) [reply less/styles.less]: Yeah, I thought so as well, but when you view ecdtools on mobile screen size, I find this style and makes it square.
image

**Pull request merged**